crates/transaction-pool/src/pool/pending.rs

```diff
@@ -34,6 +34,11 @@ pub(crate) struct PendingPool<T: TransactionOrdering> {
     by_id: BTreeMap<TransactionId, PendingTransaction<T>>,
     /// _All_ transactions sorted by priority
     all: BTreeSet<PendingTransaction<T>>,
+    /// The highest nonce transactions for each sender - like the `independent` set, but the
+    /// highest instead of lowest nonce.
+    ///
+    /// Sorted by their scoring value.
+    independent_descendants: BTreeSet<PendingTransaction<T>>,
```

Comment:
hmm,the naming here is confusing, because these are only independent if the account only has a single pending tx?

Reply:
yeah, this should be renamed, they aren't independent